.gitignore

```diff
@@ -102,3 +102,5 @@ ENV/
 
 #dynaconf settings
 /settings.yml
+5488499.png
```

Comment:
I think you've added this entries while testing locally. Please remove since they do no apply for every contributor.